### PR TITLE
Updated heading title

### PR DESCRIPTION
**Pull request by ianf77**
Changed Migration to Preparing to migrate

Bug 2013679 - Migration pulp2 to pulp3 a pre-migration or a migration

https://issues.redhat.com/browse/SATDOC-491

Cherry-pick into:

* [X] Foreman 3.0
* [X] Foreman 2.5 (Satellite 6.10)

### CODE REVIEW

**Review by maximiliankolb — COMMENTED** [review]
Fine for me; ideally you also adapt the file name and anchor (and replace old xrefs if necessary).

File: guides/doc-Upgrading_and_Updating/topics/migrating_pulp_content.adoc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [id="migrating_pulp_content"]
 
-= Migrating Content to Pulp 3
+= Preparing to Migrate Content to Pulp 3
```
Comment (lzap) [inline guides/doc-Upgrading_and_Updating/topics/migrating_pulp_content.adoc:3]: Isn't Preparing Pulp 3 Content Migration better? @melcorr 

Please update the anchor (the id above). Should be in this format ideally: `Preparing_Blah_Blah`

**Comment by lzap** [comment]
You missed one mor:

```
[lzap@nuc foreman-documentation]$ ag migrating_pulp_content
guides/doc-Upgrading_and_Updating/topics/migrating_pulp_content.adoc
1:[id="migrating_pulp_content"]

guides/doc-Upgrading_and_Updating/topics/upgrading_satellite_server_parent.adoc
10:You can reduce this time by following the procedure in xref:migrating_pulp_content[]

guides/doc-Upgrading_and_Updating/topics/upgrading_satellite_server_prerequisites.adoc
55:For further information, see xref: migrating_pulp_content[Migrating Content to Pulp 3].

guides/doc-Upgrading_and_Updating/master.adoc
32:include::topics/migrating_pulp_content.adoc[leveloffset=+3]
```

**Review by lzap — APPROVED** [review]
Sure, thanks.

**Comment by lzap** [comment]
027e0a76..9886d78b  2.5 -> 2.5

**Pull request merged**